### PR TITLE
Javascript quize - added two new questions

### PR DESCRIPTION
**Pull request by isurumaldeniya**
Checklist:

- [ x ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ x ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

**Comment by jdwilkin4** [comment]
Please make sure not to include the `Closes #14 ` because this issue is supposed to stay open for multiple contributors.

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1158,7 +1158,25 @@ const javascriptQuiz = [
...
     Link: "https://www.freecodecamp.org/news/how-to-insert-an-element-into-an-array-in-javascript/"
-  }
+  },
+  {
+    Question: "In JavaScript, from following what isn't use to declare a variable?",
+    Answer: "number",
+    Distractor1: "let",
+    Distractor2: "var",
+    Distractor3: "const",
+    Explanation: "JavaScript has three kinds of variable declarations. var declare a variable, optionally initializing it to a value, let declare a block scoped, local variable,optionally initializing it to a value and const declare read-only block scoped constant",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1168]: Let's trim down this quiz question a bit. If users want to learn more they can click on the learn more link to read the article. 

```suggestion
    Explanation: "JavaScript has three kinds of variable declarations: var, let and const.",
```

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -1158,7 +1158,25 @@ const javascriptQuiz = [
     Explanation:
       ".push() returns the new length of the array, after the element has been pushed",
     Link: "https://www.freecodecamp.org/news/how-to-insert-an-element-into-an-array-in-javascript/"
-  }
+  },
+  {
+    Question: "In JavaScript, from following what isn't use to declare a variable?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1163]:
```suggestion
    Question: "In JavaScript, which of the following is NOT the correct way to declare a variable?",
```

File: src/data/javascript-quiz.ts [hunk 3]
```diff
@@ -1158,7 +1158,25 @@ const javascriptQuiz = [
...
+    Explanation: "JavaScript has three kinds of variable declarations. var declare a variable, optionally initializing it to a value, let declare a block scoped, local variable,optionally initializing it to a value and const declare read-only block scoped constant",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_Types#declarations",
+  },
+  {
+    Question: "In JavaScript, how many primitive data types are there?",
+    Answer: 7,
+    Distractor1: 3,
+    Distractor2: 6,
+    Distractor3: 8,
+    Explanation: "JavaScript has 7 primitive data types. Boolean, null, undefined, Number, BigInt, String, Symbol",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1177]:
```suggestion
    Explanation: "JavaScript has 7 primitive data types. Boolean, null, undefined, Number, BigInt, String, and Symbol",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by isurumaldeniya** [comment]
@jdwilkin4 Hi I have done the changes please review the code and let me know if anything needed to be changed.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by isurumaldeniya** [comment]
@jdwilkin4 Thank you for the guidance throughout the PR. I really hope to contribute more to the freeCodeCamp in the future. 😄

**Pull request merged**